docs/publishing/publishing.md

```diff
@@ -27,6 +27,17 @@ You will also have the option to create a new draft based on the currently publi
 
 Note that you cannot edit an unpublished draft created by another user.
 
+### Autosave
+
+eZ Platform comes with an Autosave feature that helps you preserve your work in an event of a failure while you edit a content item or product description.
+To recover your work, open the most recent draft in the **Versions** tab of the content item or
+product description.
```

Comment:
```suggestion
To recover your work, open the most recent draft in the **Versions** tab of the Content item.
```
If product is stored in the content tree, it is a Content item, no need to mention it explicitly.